StoreManagementService/src/PBJ.StoreManagementService.Business/Services/UserService.cs

```diff
@@ -109,8 +109,17 @@ public async Task<bool> DeleteAsync(int id)
 
             await _userRepository.DeleteAsync(existingUser);
 
-            await _userFollowersRepository.DeleteRangeAsync(existingUser.Followers);
-            await _userFollowersRepository.DeleteRangeAsync(existingUser.Followings);
+            existingUser.Followers = new List<UserFollowers>();
+
+            if (existingUser.Followers != null)
+            {
+                await _userFollowersRepository.DeleteRangeAsync(existingUser.Followers);
+            }
+
+            if (existingUser.Followings != null)
+            {
+                await _userFollowersRepository.DeleteRangeAsync(existingUser.Followings);
+            }
 
             return true;
         }
```

Comment:
Not an issue, just a question, what is the reason to return that flag from the `delete` method?